packages/app-scripts-preact/package.json

```diff
@@ -12,9 +12,12 @@
     "@babel/preset-env": "^7.9.5",
     "@babel/preset-react": "^7.9.4",
     "@babel/preset-typescript": "^7.9.0",
+    "@prefresh/snowpack": "0.1.0",
     "@snowpack/babel-plugin-package-import": "^0.6.1",
+    "@snowpack/plugin-babel": "^0.6.1",
     "babel-jest": "^25.4.0",
-    "react-app-polyfill": "^1.0.6"
+    "react-app-polyfill": "^1.0.6",
+    "react-refresh": "^0.8.2"
```

Comment:
just curious how this relates to react-refresh, can you explain this a bit?

Reply:
I think this is needed for the babel plugin for hooks. It's located at `react-refresh/babel`